src/armor_layers.cpp

```diff
@@ -393,8 +393,8 @@ void player::sort_armor()
                 if( rightListSize >= rightListOffset && pos <= cont_h - 2 ) {
                     trim_and_print( w_sort_right, pos, 2, right_w - 4, elem.damage,
                                     elem.name.c_str() );
-                    mvwprintz( w_sort_right, pos, right_w - 2, c_ltgray, "%d",
-                               elem.encumber );
+                    mvwprintz( w_sort_right, pos, right_w - 3, c_ltgray, "%*d",
+                               3, elem.encumber );
```

Comment:
Out of curiosity: why the separate argument instead of putting the number into the format string as "%3d"?

Reply:
Ah didn't know it could be in there, took it from another place in the code that used *